hack/quick_install.sh

```diff
@@ -115,7 +115,7 @@ helm install \
 echo "😀 Successfully installed Cert Manager"
 
 # Install Knative
-if [ $deploymentMode = "Serverless" ]; then
+if [ "${deploymentMode}" = "Serverless" ]; then
```

Comment:
To me, the original one is ok except the `=`.
Could you please check this? 
```suggestion
if [ "${deploymentMode}" == "Serverless" ]; then
```

Reply:
@Jooho, in bash test, `=` is used as string comparison operator, and it's safe for both `[ ]` and `[[ ]]` test command. In contrast `==` is not safe for POSIX standard bash. The root cause here is that the lhs operand is not set, so the command is expanded to : `if [ = "Severless"]`.  See the test:

```bash
bash-3.2$ if [ $deploymentMode = "Serverless" ]; then echo "yes"; else echo "no"; fi
bash: [: =: unary operator expected
no
bash-3.2$ if [ $deploymentMode == "Serverless" ]; then echo "yes"; else echo "no"; fi
bash: [: ==: unary operator expected
no
bash-3.2$ export deploymentMode=local
bash-3.2$ if [ $deploymentMode == "Serverless" ]; then echo "yes"; else echo "no"; fi
no
bash-3.2$ unset deploymentMode
bash-3.2$ echo $deploymentMode

bash-3.2$ if [ "${deploymentMode}" == "Serverless" ]; then echo "yes"; else echo "no"; fi
no
```